### ISSUE TITLE
Latest update no longer loads indexes

### ISSUE THREAD

**Issue by AdrianMF** [AntonShuvaev/elasticsearch4idea#35]
I just updated the plugin yesterday and it seems something has been broken. When I go to the indexes, its just blank and no amount of refreshing seems to load them back:
![Screen Shot 2021-06-17 at 10 32 22 AM](https://user-images.githubusercontent.com/1189227/122406680-68114a00-cf57-11eb-89c6-df4e8165861a.png)
![Screen Shot 2021-06-17 at 10 33 23 AM](https://user-images.githubusercontent.com/1189227/122406791-80816480-cf57-11eb-8a4d-4fc42d31159f.png)
![Screen Shot 2021-06-17 at 10 33 44 AM](https://user-images.githubusercontent.com/1189227/122406804-82e3be80-cf57-11eb-816c-5a5783dbeafc.png)

**Comment by AntonShuvaev** [comment]
Hi!

It seems like a bug that the aliases folder is not empty when cluster is disconnected. But I cannot reproduce the issue with empty indices. Can you try to `Test Connection` in `Edit Elasticsearch Cluster` window?

**Comment by AdrianMF** [comment]
Looks good to me:
![Screen Shot 2021-06-17 at 10 49 32 AM](https://user-images.githubusercontent.com/1189227/122409799-cf2ffe00-cf59-11eb-901f-22d238ababfe.png)
It loads the correct aliases:
![Screen Shot 2021-06-17 at 10 50 32 AM](https://user-images.githubusercontent.com/1189227/122409934-e8d14580-cf59-11eb-9675-da72d1a683fb.png)

**Comment by AntonShuvaev** [comment]
It seems like a bug. I will investigate it. Can you try to edit cluster properties and save them? If it doesn't help try to restart IDE.

**Comment by AdrianMF** [comment]
I tried a few permutations of editing the cluster then created a cluster from scratch and restarted a few times with no progress on the issue.

**Comment by AntonShuvaev** [comment]
Which version of Elasticsearch do you use?

**Comment by AdrianMF** [comment]
```json
  "version" : {
    "number" : "7.4.2",
    "build_flavor" : "oss",
    "build_type" : "tar",
    "build_hash" : "unknown",
    "build_date" : "2020-10-15T22:39:41.430997Z",
    "build_snapshot" : false,
    "lucene_version" : "8.2.0",
    "minimum_wire_compatibility_version" : "6.8.0",
    "minimum_index_compatibility_version" : "6.0.0-beta1"
  }
```

**Comment by AntonShuvaev** [comment]
It seems like the bug related to fetching nodes info. Could you share the response of request `{host}/_cat/nodes?v&full_id&h=http_address,ip,heap.percent,ram.percent,cpu,load_1m,load_5m,load_15m,node.role,master,name,id,version,disk.used,disk.avail`? This will really help me because I cannot reproduce it on my local machine.

**Comment by AdrianMF** [comment]
![Screen Shot 2021-06-17 at 1 36 42 PM](https://user-images.githubusercontent.com/1189227/122438476-2392a800-cf71-11eb-8b87-322f23969831.png)

**Comment by AntonShuvaev** [comment]
@AdrianMF, thanks. I fixed this issue in 1.4.1 that will be available soon. Let me know if the issue will still persist.

**Comment by AdrianMF** [comment]
Its working now thanks!
